src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -88,7 +82,8 @@ def start(scenario, settings, events, aggregator_device_mapping):
             "max_panel_power_W": settings.get('max_panel_power_W',
                                               ConstSettings.PVSettings.MAX_PANEL_OUTPUT_W),
             "grid_fee_type": settings.get('grid_fee_type', GlobalConfig.grid_fee_type),
-            "external_connection_enabled": True
+            "external_connection_enabled": True,
+            "aggregator_device_mapping": aggregator_device_mapping
```

Comment:
This will not work, `aggregator_device_mapping ` is not defined, because you deleted line 65-68.

Reply:
sorry, forget it